generate/templates/embed_{{.os}}_{{.arch}}.go.tpl

```diff
@@ -0,0 +1,25 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+// generated by github.com/apache/opendal-go-services
+
+package {{.pkg}}
+
+import _ "embed"
+
+//go:embed libopendal_c.so.{{.os}}.{{.arch}}.zst
```

Comment:
I personally feel like file name should be `libopendal_c.{{.os}}.{{.arch}}.so.zst` to avoid confusing.

Reply:
make sense